pom.xml

```diff
@@ -667,18 +674,19 @@
                   </bind>
                 </volumes>
                 <env>
+                  <LAMBDA_S3_BUCKET>${lambda.s3.bucket}</LAMBDA_S3_BUCKET>
                   <BUCKETEER_S3_BUCKET>${bucketeer.s3.bucket}</BUCKETEER_S3_BUCKET>
                   <BUCKETEER_S3_REGION>${bucketeer.s3.region}</BUCKETEER_S3_REGION>
                   <BUCKETEER_S3_ACCESS_KEY>${bucketeer.s3.access_key}</BUCKETEER_S3_ACCESS_KEY>
                   <BUCKETEER_S3_SECRET_KEY>${bucketeer.s3.secret_key}</BUCKETEER_S3_SECRET_KEY>
                   <BUCKETEER_IIIF_URL>${bucketeer.iiif.url}</BUCKETEER_IIIF_URL>
                   <SLACK_TEST_USER_HANDLE>${bucketeer.slack.test_user_handle}</SLACK_TEST_USER_HANDLE>
-                  <SLACK_OAUTH_TOKEN>${bucketeer.slack.oauth_token}</SLACK_OAUTH_TOKEN>
-                  <SLACK_ERROR_CHANNEL_ID>${bucketeer.slack.error_channel_id}</SLACK_ERROR_CHANNEL_ID>
-                  <SLACK_CHANNEL_ID>${bucketeer.slack.channel_id}</SLACK_CHANNEL_ID>
-                  <BUCKETEER_FS_PREFIX>GenericFilePathPrefix</BUCKETEER_FS_PREFIX>
-                  <BUCKETEER_FS_MOUNT>/images/</BUCKETEER_FS_MOUNT>
-                  <FEATURE_FLAGS>https://iiif-manifest-store.s3.amazonaws.com/bucketeer-features-off.conf</FEATURE_FLAGS>
+                  <BUCKETEER_SLACK_OAUTH_TOKEN>${bucketeer.slack.oauth_token}</BUCKETEER_SLACK_OAUTH_TOKEN>
+                  <BUCKETEER_SLACK_ERROR_CHANNEL_ID>${bucketeer.slack.error_channel_id}</BUCKETEER_SLACK_ERROR_CHANNEL_ID>
+                  <BUCKETEER_SLACK_CHANNEL_ID>${bucketeer.slack.channel_id}</BUCKETEER_SLACK_CHANNEL_ID>
+                  <BUCKETEER_FS_IMAGE_PREFIX>GenericFilePathPrefix</BUCKETEER_FS_IMAGE_PREFIX>
+                  <BUCKETEER_FS_IMAGE_MOUNT>/images/</BUCKETEER_FS_IMAGE_MOUNT>
+                  <FEATURE_FLAGS>https://iiif-manifest-store.s3.amazonaws.com/test-bucketeer-features-off.conf</FEATURE_FLAGS>
```

Comment:
Note that these tests are now using a feature flag config that I created specifically for testing. I was hesitant to change the config file that was being used in production.

Reply:
@cachemeoutside has suggested using config files in the github repo. Another bucket could also be used. Let's also talk about this one tomorrow in the zoom room.